doc/news/changes.h

```diff
@@ -540,6 +540,13 @@ inconvenience this causes.
 
 
 <ol>
+  <li> New: VectorizedArray now provides two methods @p
+  vectorized_load_and_transpose and @p vectorized_transpose_and_store that
+  perform vectorized reads or writes and convert from array-of-struct into
+  struct-of-array or the other way around.
+  <br>
```

Comment:
The linkable reference from Doxygen reads VectorizedArray::vectorized_load_and_transpose() without the `@p`